integration-tests/plugins/__tests__/regions/admin/regions.spec.ts

```diff
@@ -63,14 +65,21 @@ describe("Regions - Admin", () => {
         id: created.data.region.id,
         name: "Test Region",
         currency_code: "usd",
+        metadata: { foo: "bar" },
       })
     )
+    expect(created.data.region.countries.map((c) => c.iso_2).sort()).toEqual([
```

Comment:
todo: can you rm the sorting everywhere as it is by default on the relation. Normally the default sql sorting should be enough

Reply:
@sradevski could you remove the sort in all the files please